README.md

```diff
@@ -26,7 +27,7 @@ inefficient code. Finding this code is usually pretty difficult.
 5. Type in function that you would like to start tracing.
 6. Start tracing clicking green button.
 
-Example rebar2 config entry:
+Example rebar3 config entry:
```

Comment:
It is not valid `rebar3` configuration entry (those are not compatible between `rebar2` and `rebar3` deliberately).